truth-android-appcompat-v7/build.gradle

```diff
@@ -17,10 +17,6 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_7
         targetCompatibility JavaVersion.VERSION_1_7
     }
-
```

Comment:
I think we still want to fail link when there is an error. I'd prefer we disable checks that produce false positives instead.

Reply:
Fixed